db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -521,8 +537,9 @@ class NonBatchedOpsStressTest : public StressTest {
           switch (op) {
             case 0:
             case 1: {
-              uint32_t value_base =
-                  thread->rand.Next() % thread->shared->UNKNOWN_SENTINEL;
+              const uint32_t value_base = ExpectedState::NextValueBase(
+                  ExpectedState::GetValueBase(thread->shared->Get(
+                      rand_column_families[0], rand_keys[i])));
```

Comment:
It'd be easier to leave this as `thread->rand.Next() % thread->shared->RANDOM_VALUE_BASE_GENERATOR`. I think the counter updates aren't needed since it's only used for Get/MultiGet txn read consistency and then the txn is rolled back

Reply:
Fixed